CMakeLists.txt

```diff
@@ -395,13 +395,9 @@ set(CMAKE_REQUIRED_INCLUDES ${CURL_INCLUDE_DIRS})
 set(CMAKE_REQUIRED_LIBRARIES ${CURL_LIBRARIES})
 CHECK_SYMBOL_EXISTS(curl_multi_wait curl/curl.h HAVE_CURL_MULTI_WAIT)
 
-find_package(NSS REQUIRED)
```

Comment:
might want to remove `cmake/modules/Find{NSS,NSPR}.cmake` accordingly.

Reply:
Thanks for pointing this out! Pushing amended commit.

Reply:
these two scripts are not removed in the latest version.

Reply:
Apologizes. My repo ended in the detached state. Repushed.